packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -543,6 +551,7 @@ export interface SdkHttpOperation extends SdkServiceOperationBase {
   __raw: HttpOperation;
   kind: "http";
   path: string;
+  uriTemplate: string;
```

Comment:
What does emitter suppose to do with `uriTemplate` currently? I think we only care about `allowReserved` and `explode` for now, this field is for the whole `uriTemplate` feature support in the future? Please correct me if I'm wrong.

Also is this a must-have field, or optional field? `uriTemplate?: string` or `uriTemplate: string`

Reply:
1. correct
2. required field (same with typespec core)

Reply:
Right now you can do whatever you want to support whatever you want. But the longer term goal is you use the uni template to produce the uri by using an implementation of the algorithm(not very complex)